lasagne/layers/recurrent.py

```diff
@@ -356,7 +364,10 @@ def get_output_for(self, inputs, **kwargs):
             part of the sequence or not.  `mask` should be a matrix of shape
             ``(n_batch, n_time_steps)`` where ``mask[i, j] = 1`` when ``j <=
             (length of sequence i)`` and ``mask[i, j] = 0`` when ``j > (length
-            of sequence i)``.
+            of sequence i)``. When the hidden state of this layer is to be
+            pre-filled (i.e. was instantiated with `isinstance(hid_init,
+            Layer)`) `inputs` should have length at least 2, and `inputs[-1]`
```

Comment:
This should probably say ` ``isinstance(hid_init, Layer) == True`` `.  Note the double backticks.